inc/class-cloner.php

```diff
@@ -570,11 +615,14 @@ protected function cloneSection( $section_id, $post_type, $parent_id = null ) {
 		foreach ( $this->sourceBookStructure['_embedded'][ $post_type ] as $k => $v ) {
 			if ( $v['id'] === absint( $section_id ) ) {
 				$section = $this->sourceBookStructure['_embedded'][ $post_type ][ $k ];
+				break;
 			}
 		};
 
-		// Get links
```

Comment:
Same as previous comment; `_links` needs to be unset.

Reply:
Oh! OK. I'll change the variable to `$_links` and leave a comment. To me it looked like a variable that was not set.